src/themes/teams/components/Chat/chatItemStyles.ts

```diff
@@ -1,25 +1,76 @@
 import { ICSSInJSStyle, ComponentSlotStylesInput } from '../../../types'
 import { ChatItemVariables } from './chatItemVariables'
 import { ChatItemProps } from '../../../../components/Chat/ChatItem'
+import { pxToRem } from '../../../../lib'
+import ChatMessage from '../../../../components/Chat/ChatMessage'
+
+const chatMessageClassname = `& .${ChatMessage.className}`
```

Comment:
can we add these values to a constants file somewhere or export them from `ChatMessage.tsx` so there's only one place we change them?

Reply:
> Last question, how do we want to export the classNames for the slots? See the author and timestamp classNames in the ChatMessage component.

what about exporting a constants object from `ChatMessage.tsx`?

```typescript
export const chatMessageMetadata = {
  authorClassName: `${ChatMessage.className}__author`,
  timestampClassname: `${ChatMessage.className}__timestamp`,
}
```